buddy_init/init_project.py

```diff
@@ -0,0 +1,55 @@
+from functools import partial
+
+import argparse
+
+from .init_actions import *
+from .init_framework import WorkingDirectory, prompt, add_action_toggle
+
+
+def init(args=None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--yes', '-y', help='Forces all boolean options to true', action='store_true', default=None)
+    parser.add_argument('--dest', '-d', help='The destination of your epic project! :D', required=True)
+    parser.add_argument('--force', '-f', help='Starts the project from scratch', action='store_true', default=False)
+
+    add_action_toggle(parser, setup_mila_user)
+    add_action_toggle(parser, create_venv)
```

Comment:
if already in venv maybe skip this?

Reply:
From the call made with Manuel:
 - if create_venv='Y': create venv
 - if create_venv='N': use existing python interpreter (unless it's the system interpreter, in which case raise Exception) 

Better to use the existing venv by default, or give the option to create a venv just for the current project?

Reply:
if already in a venv, this should be skipped imo. @manuel-delverme ?

Reply:
maybe be like:
skipping, you already have an env, if you want to use another deactivate and restart. Enter to continue.

so people can't fuckit up